NRFX-3926: Add links to pmic documentation (#43)
NordicSemiconductor/npmx-zephyr: README.md

>export ZEPHYR_BASE=<path_to_zephyr_directory>
>```

> In order to run a custom application please refer to the [documentation][npm1300_documentation] to learn how to properly configure the required setup.

Reviewer: I see, let's try this way:
```suggestion
If you want to run a custom application, refer to the [nPM1300 documentation][nPM1300_documentation] to learn how to properly configure the required setup.
```

> If you want to run a custom application, refer to the [nPM1300 documentation][nPM1300_documentation] to learn how to properly configure the required setup.

Reviewer 2: Perhaps a link to npmx docs should be also added here? How about:
`In order to create a custom application, refer to npmx docs for API usage and nPM1300 documentation for HW setup and PMIC HW functionalities.`

Reviewer: ```suggestion
If you want to create a custom application, refer to the following guidelines:
- [link to npmx documentation] for API usage
- [nPM1300 documentation][nPM1300_documentation] for the required setup configuration
```

```

[doc_dir]: doc

> [npm1300_documentation]: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html

Reviewer: ```suggestion
[nPM1300_documentation]: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_pmic%2Fstruct%2Fnpm1300.html
```
